### DISCUSSION TITLE
Assessment Framework options

### DISCUSSION THREAD

**Discussion by wcoreiron** [prowler-cloud/prowler#4022] [Q&A]
I had a chance to assess Azure environment and I got CIS outputs. I wanna get NiSt or or SOC2 outputs. How can I get these frameworks as well. Thanks!

**Comment by jfagoagas** [comment]
Hi @wcoreiron actually for the Azure provider we cover CIS 2.0, CIS 2.1 and Mitre. We are working towards covering more and more compliance frameworks like NIST or SOC2. Stay tuned for the next releases!

Thanks for using Prowler 🚀

**Reply by DrSquare86** [reply]
keen on seeing Nist and SOC2 any updates on this?

**Reply by jfagoagas** [reply]
They are still on our internal roadmap. It'd be great if you can open a new `feature-request` [here](https://github.com/prowler-cloud/prowler/issues?q=sort:updated-desc+is:issue+is:open).

Thanks!
